src/libs/modulelist.c

```diff
@@ -366,10 +366,79 @@ void init_presets(dt_lib_module_t *self)
   dt_lib_presets_add(_("show all"), self->plugin_name, self->version(), params_all, len);
 
   /* the modules that are activated by default in the initial configuration */
-  dt_lib_presets_add(_("default"), self->plugin_name, self->version(), params_default, len);
+  dt_lib_presets_add(_("default"), self->plugin_name, self->version(), params, len);
+
+  params = gen_params(1, &len,
+                        "|demosaic|exposure|colorin|temperature|clipping"           // raw handling
+                        "|filmic|tonecurve|levels"                                  // tones
+                        "|channelmixer|colorbalance|colorzones|colorchecker|vibrance" // colors
+                        "|sharpen|bilat|atrous"                                     // sharpness
+                        "|cacorrect|defringe|highlights|lens|colorreconstruction"   // image reconstruction
+                        );
+  dt_lib_presets_add(_("0. all-purpose toolbox"), self->plugin_name, self->version(), params, len);
```

Comment:
What are the numbers meaning here? "0." above then "1."... "I.", "II." ?

Is that needed? Sounds like noise to me.

Reply:
The arabic numbers are for toolboxes, which are aimed at providing a complete workspace. The roman numbers are for modules subsets, which are aimed at exposing quickely some features (purely technical and purely creative modules), but are not full workspaces.

Reply:
I see, let me try this then.

Reply:
Sorry don't like the number because they do not fit well with others preset like all or none activated.

What about:

toolbox : all purpose
toolbox : portrait & beauty
...

subset: creative only
subset: technical only

and the current presets:

subset: all
subset: none
subset: default

We have now two groups, sorted and named.

Reply:
that's another option indeed

Reply:
So if ok with you, let's go this way. Thanks.